packages/queued-request-controller/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@metamask/queued-request-controller",
+  "version": "1.0.0",
```

Comment:
I think we ought to leave this at `0.0.0`. If we do this, then this reduces the risk that this package will get released accidentally, and it will also be more clear that we are releasing a new version because we will bump this in the release PR. Does that make sense?